src/database.c

```diff
@@ -469,6 +508,7 @@ _add_to_db(ProfMessage* message, char* type, const Jid* const from_jid, const Ji
                                               date_fmt ? date_fmt : "",
                                               message->id ? message->id : "",
                                               message->stanzaid ? message->stanzaid : "",
+                                              original_message_id == -1 ? "" : sqlite3_mprintf("%d", original_message_id),
```

Comment:
Quoting the sqlite documentation:

> The sqlite3_mprintf() and sqlite3_vmprintf() routines write their
** results into memory obtained from [sqlite3_malloc64()].
** The strings returned by these two routines should be
** released by [sqlite3_free()]

Reply:
Thanks, done.